discord/flags.py

```diff
@@ -39,6 +39,7 @@
     overload,
 )
 
+import discord
```

Comment:
Why are you directly importing the `discord` namespace? Just import what you need